### DISCUSSION TITLE
PRIDE Submission Tool stuck at checksum stage

### DISCUSSION THREAD

**Discussion by ypriverol** [PRIDE-Archive/pride-resources#3] [Q&A]
If you don’t have write access to the folders you are running the tool the checksum generation step can be stcuk. 
You can check by creating a file in that folder and check. For example, if you have the submission tool on the C drive with Windows 10, in the Program Files directory, that is apparently protected, and the installation tool cannot write there. If you put the installation tool into another directory with no read/write permissions (apparently), then it all works fine. I noticed when starting the tool when it was in the directory that could not write, there was an error box when the tool first started, but there was no text in that box or indication what the error was. I supposed that could be enhanced to indicate to people that write permissions are such that the upload tool cannot work.

**Comment by vbrennsteiner** [comment]
Follow up on this, as I have been facing similar issues: when calculating a checksum.txt file with the instructions provided here: https://www.ebi.ac.uk/pride/markdownpage/checksum, the submission tool freezes after selecting the checksum.txt file. Unfortunately, the tool also freezes when calculating checksums for large file numbers, so I was wondering if there is an alternative way to incorporate checksum.txt into an upload? 

Thank you very much :)
